tutorials/plot_introduction.py

```diff
@@ -130,7 +130,7 @@
 #
 # Read data from file:
 
-raw = mne.io.Raw(raw_fname)
+raw = mne.io.read_raw_fif(raw_fname)
```

Comment:
(maybe you missed them because `flake` didn't tell you about them? I suspect this is the case -- if so we need to update `flake` tests)